### PR TITLE
Update redux to 1.0.0-rc (fixes #191)

### LINKED ISSUE #191: Consider updating to Redux 1.0 RC
Hey, cheers for using Redux! :-P

Redux 1.0 RC is out with breaking changes. **No further API changes are expected in 1.0 so it may be a good time to port the repo to 1.0 RC (in a branch).** We will release 1.0 when the docs are ready.

You can find the comprehensive list of the changes in 1.0 RC against 1.0 alpha in the **[Redux 1.0 RC release notes](https://github.com/gaearon/redux/releases/tag/v1.0.0-rc)**. If you haven't ported to alpha yet, you'll need to **[apply changes from 0.12 to alpha first](https://github.com/gaearon/redux/pull/195#issue-92206321)**.

Thank you!
Please make sure to report any issues so we can release a stable 1.0.

### PR DESCRIPTION
**Pull request by muffinresearch**

### CODE REVIEW

**Comment by muffinresearch** [comment]
There's still some work to do on this before it's ready...

File: public/js/apps/payment/app.jsx [hunk 1]
```diff
@@ -2,7 +2,7 @@ import 'shims';
 
 import React, { Component, PropTypes } from 'react';
 
-import { Connector, Provider } from 'redux/react';
+import { Connector, Provider } from 'react-redux';
 import { bindActionCreators } from 'redux';
 
 import reduxConfig from 'redux-config';
```
Comment (kumar303) [inline public/js/apps/payment/app.jsx]: maybe we should call this: `dataStore from 'data-store'` ?

Comment (kumar303) [inline public/js/apps/payment/app.jsx]: or `redux-store` ? "Config" no longer makes sense

**Comment by kumar303** [comment]
wow, I didn't know the upgrade was this simple. Cool.

r+wc

**Pull request merged**